### ISSUE TITLE
Extra Tests

### ISSUE THREAD

**Issue by mkduer** [mkduer/reinforcement-learning-snake-game#76]
- [x] given extra time run 1,000,000+ episodes to see just how well the snake can learn (e.g. in terms of peak score, in terms of game converging) 

- [x] adjusting board size (larger, smaller, regular)

- [x] reward tests #73 

- [x] test vs training: run test on constant hyperparameters multiple times to check for the consistency and overfitting (note append to file for this), possibly add with a changed board size

---
Discarded Tests:
  - [x] ~~randomized mouse~~ approaches were more complicated than expected, so decided to forgo this for now
  - [x] ~~increased penalty for going back on itself -20~~ should be factored in with modified state representation, and seemed more fun to focus on other tests given limited time
  - [x] ~~start with Epsilon = 0.1 and gradually increase over the training~~ to see how the exploration over to exploitation affects the Snake's performance
